### PR TITLE
feature/TASK-1_setup-project

### PR DESCRIPTION
**Pull request by milkanuch**
- [x] Add alias paths
- [x] Add ESLint rules
- [x] Add prettier
- [x] Add ESLint / prettier packages

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        'external',
+        'parent',
+        'sibling',
+        'internal',
+        'index',
+        'object',
+      ],
+      pathGroups: [
+        {
+          pattern: '{react,react-native,recoil,recoil-nexus,}',
```
Comment (JeriRov) [inline .eslintrc.js:65]: I think the comma is redundant here after the recoil-nexus

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+          group: 'parent',
+          position: 'before',
+        },
+        {
+          pattern: '{assets/**,constants/**}',
+          group: 'parent',
+          position: 'before',
+        },
+        {
+          pattern: '{helpers/**,services/**,navigators/**,store/**,}',
```
Comment (JeriRov) [inline .eslintrc.js:85]: here, too, for beauty, you can remove the comma after store/**

**Review by JeriRov — CHANGES_REQUESTED** [review]
Remove extra commas, in those places where I noticed

File: .eslintrc.js (hunk 1 above)
Comment (milkanuch) [reply .eslintrc.js:65]: As ESLintrc has a JS extension, we follow our ESLint rules in this file, we have prettier regulations, and one of these rules is about the last comma in the object.

[rule](https://github.com/prettier/eslint-plugin-prettier#options):

File: .eslintrc.js (hunk 2 above)
Comment (milkanuch) [reply .eslintrc.js:85]: As ESLintrc has a JS extension, we follow our ESLint rules in this file, we have prettier regulations, and one of these rules is about the last comma in the object.

[rule](https://github.com/prettier/eslint-plugin-prettier#options):

**Review by JeriRov — APPROVED** [review]

**Pull request merged**